elisp-bytecode.texi

```diff
@@ -79,7 +79,7 @@ of the stack.
 
 @subsubsection Example
 
-With dynamic binding is in effect, @code{(defun en(n5) (setq n 5)} generates:
```

Comment:
"When", or omit the "is", I think.

Reply:
Ok. But just to be clear since I don't know, does this matter whether lexical or dynamic binding is in effect?  There was a typo in the parameter in an earlier example where I had `n5` where I meant `n` (and `n5 was supposed to be the function name).

Reply:
I'm not sure myself. I think the problem is we're disabling optimization in the disassembler, and I'm not sure whether eliding varrefs to lexical variables counts as an optimization.

Reply:
Oh, wait, I was mis-using `eval`.  Passing `t` as the `lexical` argument, I get:

```
With dynamic binding is in effect, @code{(defun n5(n) (setq n 5))} generates:
@verbatim
PC  Byte  Instruction
 0  192   (byte-constant 5)
 1  137   (byte-dup . 0)
 2  178   (byte-stack-set . 2)
           2
 4  
135   (byte-return . 0)
@end verbatim
```

Reply:
So, yes, it does matter. Apart from the typo, and one "the the", the changes look good to me here!

Reply:
How does one use `byte-pretty` to force lexical binding?

Reply:
Sorry, just working on that. I think it's best just to accept any line with "lexical" in it to switch to lexical binding, but I'm open to other suggestions (of course the line then ends up being too long and my regexp-fu fails because it doesn't match newlines...)

Reply:
This is perfectly fine. Beggars can't be choosers.